src/views/notifications/Create.vue

```diff
@@ -0,0 +1,323 @@
+<template>
+  <div>
+    <div class="grid grid-cols-12 gap-6 mt-8">
+      <div class="col-span-12 lg:col-span-12 xxl:col-span-12">
+        <!-- BEGIN: Notification Page -->
+        <Header></Header>
+        <div class="intro-y chat grid grid-cols-12 gap-5 mt-5">
+          <!-- BEGIN: Notification Side Menu -->
+          <div class="col-span-12 lg:col-span-4 xxl:col-span-3 chat__box overflow-y-auto">
+            <div class="tab-content">
+              <div class="pr-1">
+                <div class="box p-5 mb-5">
+                  <div class="relative text-gray-700 dark:text-gray-300">
+                    <input
+                      type="text"
+                      class="form-control py-3 px-4 border-transparent bg-gray-200 pr-10 placeholder-theme-13"
+                      placeholder="Search user..."
```

Comment:
"Search **for** user"

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/34839e538cf6eea37658b6eebcf618108f1b8fc8